src/arch/ArchHooks/ArchHooks_Win32.cpp

```diff
@@ -150,26 +150,13 @@ void ArchHooks_Win32::SetTime( tm newtime )
 
 void ArchHooks_Win32::BoostPriority()
 {
-	/* We just want a slight boost, so we don't skip needlessly if something happens
-	 * in the background.  We don't really want to be high-priority--above normal should
-	 * be enough.  However, ABOVE_NORMAL_PRIORITY_CLASS is only supported in Win2000
-	 * and later. */
-#ifndef ABOVE_NORMAL_PRIORITY_CLASS
-#define ABOVE_NORMAL_PRIORITY_CLASS 0x00008000
-#endif
-
-	DWORD pri = HIGH_PRIORITY_CLASS;
-	if( IsWindowsVersionOrGreater(HIBYTE(_WIN32_WINNT_WIN2K), LOBYTE(_WIN32_WINNT_WIN2K), 0) )
-		pri = ABOVE_NORMAL_PRIORITY_CLASS;
-
-	/* Be sure to boost the app, not the thread, to make sure the
-	 * sound thread stays higher priority than the main thread. */
-	SetPriorityClass( GetCurrentProcess(), pri );
+	// Be sure to boost the app, not the thread, to make sure the sound thread stays higher priority than the main thread.
+	SetPriorityClass( GetCurrentProcess(), HIGH_PRIORITY_CLASS );
 }
 
 void ArchHooks_Win32::UnBoostPriority()
 {
-	SetPriorityClass( GetCurrentProcess(), NORMAL_PRIORITY_CLASS );
+	SetPriorityClass( GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS );
```

Comment:
Looking at when UnBoostPriority is called, and it seems like it's either when the user has alt+tabbed, or if they are quitting the game. Do we still want it to be ABOVE_NORMAL in those cases?

Reply:
I feel that the game's timing is not affected as badly when this is ABOVE_NORMAL, and builds I've distributed have it unboosted at ABOVE_NORMAL and I have not seen other issues. With this, I have done a lot of alt-tabbing in and out of the game and it seems to hold up on sync well.

Reply:
If someone doesn't have the app priority, I think it doesn't matter if the game's timing is off. When we regain the priority and boost it back up to high priority then the ragetimer should lock the timing back in iiuc.